src/server/editorServices.ts

```diff
@@ -192,6 +192,10 @@ namespace ts.server {
             return this.roots.map(root => root.fileName);
         }
 
+        getScriptKind() {

```

Comment:
Why does it always return Unknown here? I may have missed the part in the original PR to master